daphne/src/audit_log.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{messages::TaskId, DapTaskConfig};
+
```

Comment:
Can you add a docucomment briefly describing what this does? IIUC the implementation of the trait is in the internal repo for now, but it would be good to at least to say what this does and provide a link to internal documentation.

Reply:
yep, I can do so after we merge this into the fork